gap/FinSetsForCAP.gi

```diff
@@ -4,13 +4,41 @@
 # Implementations
 #
 
-BindGlobal( "FinSets", CreateCapCategory( "FinSets" ) );
-
-SetIsElementaryTopos( FinSets, true );
-
-AddObjectRepresentation( FinSets, IsFiniteSet and HasAsList and HasLength );
+##
+InstallMethod( CategoryOfFinSets,
+               [ ],
+               
+  function ( )
+    local overhead_option, finalize_option, FinSets;
+    
+    overhead_option := CAP_INTERNAL_RETURN_OPTION_OR_DEFAULT( "overhead", true );
```

Comment:
Just a note for the future: since options are available all the way down the function stack, "unpacking" this option here and pushing it again in line 18 is not needed.

I will fix this here and in `SkeletalFinSets` after the merge.